handlers/otel_handler.go

```diff
@@ -103,90 +102,102 @@ func (handler *otelHandler) handleEvent(event assemblers.Event) {
 	}
 }
 
-func (handler *otelHandler) createHTTPSpan(event *assemblers.HttpEvent, startTime, endTime time.Time, attrs []attribute.KeyValue) {
+func (handler *otelHandler) createHTTPSpan(event *assemblers.HttpEvent, startTime, endTime time.Time, incomingAttrs []attribute.KeyValue) {
 	var spanName string
 	if event.Request() == nil {
 		spanName = "HTTP"
 	} else {
-		spanName = fmt.Sprintf("HTTP %s", event.Request().Method)
+		spanName = event.Request().Method
```

Comment:
[HTTP span name should be one of the recognized methods](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#name). The literal string "HTTP" is only used as a fallback name with the method in the request is something non-standard.